src/components/data/SelectionDrawer.js

```diff
@@ -89,17 +89,20 @@ function SelectionToolbar(props) {
 
     let invalidTotal = 0;
     if (ResourceTypes.FOLDER === acceptedType && !multiSelect) {
+        const selectedInfoType = selectedResources[0]?.infoType;
```

Comment:
A comment might be good here - when I was first reviewing this, I was curious why we're checking for HT path list files if the user is supposed to select a single folder.